LTO does not work on OpenIndiana
ld: fatal file /var/tmp//ccEGae70.ltrans1.ltrans.0: open failed: No such file or directory

--- freerdp-3.22.0/cmake/CommonConfigOptions.cmake.~1~
+++ freerdp-3.22.0/cmake/CommonConfigOptions.cmake
@@ -19,7 +19,7 @@
 
 # known issue on android, thus disabled until we support newer CMake
 # https://github.com/android/ndk/issues/1444
-if(NOT ANDROID OR ("${CMAKE_VERSION}" GREATER_EQUAL "3.20.0"))
+if(NOT (${CMAKE_SYSTEM_NAME} MATCHES "SunOS") AND (NOT ANDROID OR ("${CMAKE_VERSION}" GREATER_EQUAL "3.20.0")))
   if(POLICY CMP0138)
     cmake_policy(SET CMP0138 NEW)
   endif()
